cloudtasks/api/HttpTasksSample/Samples.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright 2018 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Cloud.Tasks.V2Beta3;
+using Google.Protobuf;
+using Google.Protobuf.WellKnownTypes;
+using System;
+
+namespace GoogleCloudSamples
+{
+    public partial class Samples
+    {
+        // [START cloud_tasks_create_http_task]
+        public static object CreateTask(
```

Comment:
According to the new rubric, the START and END tags should enclose the using statements at the top and the complete class.